xrd_simulator/templates.py

```diff
@@ -168,10 +171,18 @@ def polycrystal_from_odf(
     )
 
     mesh = TetraMesh._build_tetramesh(cylinder)
-
-    # Sample is uniformly single phase
-    phases = [Phase(unit_cell, sgname, path_to_cif_file)]
-    element_phase_map = np.zeros((mesh.number_of_elements,)).astype(int)
+    
```

Comment:
This looks solid to me. :=)

Altough, we need [to make the unit tests pass](https://github.com/FABLE-3DXRD/xrd_simulator/blob/21e58b6b3f83bb174046416791e057b92b459e68/tests/test_templates.py#L70) / add one for the case of multi phase.